inject-groovy/src/main/groovy/io/micronaut/ast/groovy/InjectTransform.groovy

```diff
@@ -976,9 +935,6 @@ class InjectTransform implements ASTTransformation, CompilationUnitAware {
                 return
             }
             AnnotationMetadata fieldAnnotationMetadata = AstAnnotationUtils.getAnnotationMetadata(fieldNode)
-            typeElementVisitors.findAll { it.matches(fieldAnnotationMetadata) }.each {
-                it.visit(fieldNode, fieldAnnotationMetadata)
-            }
```

Comment:
There seem to be multiple changes to `InjectTransform` that are unrelated. Please rebase and remove these changes

Reply:
I must have picked these up from a merge from master? I am not sure.

Reply:
weird this case shows as matching up with master 100%

Reply:
hmm I'm having some issues rebasing.. I takes me through hundreds of other commits then dies :(